DungeonMaster/Assets/Scripts/HomeUIManager.cs

```diff
@@ -19,10 +19,12 @@ public class HomeUIManager : MonoBehaviour
     public GameObject tutorialPanel;
 
     public GameObject mainPanel;
+
+    public GameObject adminPanel;
     #endregion
 
     //TODO: Extrapolate these out to a constants file or URL builder
-    const string baseURL = "dungeonmaster-development.vapor.cloud";
+    public string baseURL = "dungeonmaster-development.vapor.cloud";
```

Comment:
Could this technically still be a const public value ?

Reply:
you right